strands_movebase/launch/obstacles.launch

```diff
@@ -66,12 +66,6 @@
         <param name="cutoff_z" value="0.3"/>
     </node>
 
-    <!-- This node removes the desired cutoff pixels from the cloud edges -->
```

Comment:
@bfalacerda  What's the deal with removing this? I can't see it launched anywhere else and if you want to use the original laser topic you'll have to change the topic in sources.